src/modules/micrortps_bridge/micrortps_client/CMakeLists.txt

```diff
@@ -84,8 +84,9 @@ if (NOT "${config_rtps_send_topics}" STREQUAL "" OR NOT "${config_rtps_receive_t
 			--mkdir-build
 			--generate-cmakelists
 			--topic-msg-dir ${PX4_SOURCE_DIR}/msg
-			--uorb-templates-dir "templates/uorb_microcdr"
-			--urtps-templates-dir "templates/urtps"
+			--uorb-templates-dir templates/uorb_microcdr
+			--urtps-templates-dir templates/urtps
+			--rtps-ids-file tools/uorb_rtps_message_ids.yaml
```

Comment:
I'd remove it since it uses the default.

Reply:
The purpose was to show an usage example. But I guess I can remove it

Reply:
done